bin/docker-entrypoint.sh

```diff
@@ -114,12 +114,12 @@ if [ "$BASE_CMD" = "odoo" ] || [ "$BASE_CMD" = "odoo.py" ]; then
     run-parts --verbose "$START_ENTRYPOINT_DIR"
   fi
 
-  if [ ! -z "$DEBUGPY_ENABLE" ]; then
+  if [ ! -z "$DEBUGPY_ENABLE" -o "$DEBUGPY_ENABLE" = True ]; then
     if [ -z "$DEBUGPY_PORT" ]; then
       DEBUGPY_PORT=8888
     fi
-    pip3 install debugpy
-    /usr/bin/python3 -m debugpy --listen 0.0.0.0:${DEBUGPY_PORT}  /usr/local/bin/odoo
+    env python3 -m pip install debugpy 
```

Comment:
```suggestion
    env python3 -m pip install debugpy pydevd-odoo
```

I recommend installing also `pydevd-odoo`, which makes the debugging experience a little better